src/backend/app/db/postgis_utils.py

```diff
@@ -785,3 +791,124 @@ def parse_featcol(features: Union[Feature, FeatCol, MultiPolygon, Polygon]):
     elif isinstance(features, Feature):
         feat_col = geojson.FeatureCollection([feat_col])
     return feat_col
+
+
+def get_osm_geometries(form_category, geometry):
+    """Request a snapshot based on the provided geometry.
+
+    Args:
+        form_category(str): feature category type (eg: buildings).
+        geometry (str): The geometry data in JSON format.
+
+    Returns:
+        dict: The JSON response containing the snapshot data.
+    """
+    config_filename = XLSFormType(form_category).name
+    data_model = f"{data_models_path}/{config_filename}.yaml"
+
+    with open(data_model, "rb") as data_model_yaml:
+        extract_config = BytesIO(data_model_yaml.read())
+
+    pg = PostgresClient(
+        "underpass",
+        extract_config,
+        auth_token=settings.RAW_DATA_API_AUTH_TOKEN
+        if settings.RAW_DATA_API_AUTH_TOKEN
+        else None,
+    )
+    return pg.execQuery(
+        geometry,
+        extra_params={
+            "fileName": (
+                f"fmtm/{settings.FMTM_DOMAIN}/data_extract"
```

Comment:
Minor thing - the folder structure starting with `fmtm/...` should only be used if we want to keep the data extract.

Raw data api has a rule if we use auth and a set structure, the file is retained for the project. 

In this case we could remove this param as we only need a temp extract 👍

Reply:
okay